Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM golang:1.13.2 as golang
 
-WORKDIR /go/src/github.com/IBM/portieris
+WORKDIR $GOPATH/src/github.com/IBM/portieris
```

Comment:
You've used `$GOPATH` here, but lines 4 and 10 refer to `/go` still. Should really be consistent.
Also, is that environment variable inherited from the (`FROM`) golang image, or do you now additionally need to pass it into the build context?

Reply:
Have undone this